amethyst_audio/src/bundle.rs

```diff
@@ -18,19 +18,15 @@ use crate::{output::Output, source::*, systems::AudioSystemDesc};
 ///
 /// The generic N type should be the same as the one in `Transform`.
 #[derive(Default, Debug)]
-pub struct AudioBundle(Output);
+pub struct AudioBundle;
 
 impl<'a, 'b> SystemBundle<'a, 'b> for AudioBundle {
     fn build(
         self,
         world: &mut World,
         builder: &mut DispatcherBuilder<'a, 'b>,
     ) -> Result<(), Error> {
-        builder.add(
-            AudioSystemDesc::new(self.0).build(world),
-            "audio_system",
-            &[],
-        );
+        builder.add_thread_local(AudioSystemDesc::new().build(world));
```

Comment:
`#[cfg(feature = "wasm")]` ?

Reply:
Hm, `rodio::OutputStream` is `!Send + !Sync`, so it can't be stored in the `World` as a resource.
I put it in the `AudioSystem` since that's where `Output` used to live (and that was `Send + Sync`).

We could require the user to instantiate `OutputDevice` outside the `Application`, and pass in the `Output` (simply a wrapper for `rodio::OutputStreamHandle` -- I tried to not break existing API too much).

Haven't actually gotten audio working in pong WASM yet, but we should decide by then what to do